### PR TITLE
Conf tt 57 implement translation engine

### PR DESCRIPTION
**Pull request by mzolovska**
**JIRA:** [🌸 TT-something 🌸](https://champlainsaintlambert.atlassian.net/jira/software/projects/TT/boards/201/backlog?selectedIssue=TT-57)

## Context:

✨ What's happening? Let's dive into the magical story! ✨
-Implemented the translation engine. Translated the current entities that we have as well as navbar and dashboard. Also set the default language to english.

## Changes:
Installed bunch of stuff in order to have the engine.
### backend:
🔧 Backend sorcery to make everything super smooth! ✨

### front end:
🌸 Added extra cuteness to the UI and made it sparkle! 🌈
Here's a small example of before and after 
image

image

## Before and After UI (Required for UI-impacting PRs):
🎀 Please attach screenshots so we can see the adorable transformation! 🐾

### CODE REVIEW

File: fe-traveltrove/23.0.1
Comment (flyindonut) [inline fe-traveltrove/23.0.1:1]: Is this empty file important?

File: fe-traveltrove/13.0.0
Comment (flyindonut) [inline fe-traveltrove/13.0.0:1]: Is this empty file important?

**Review by flyindonut — APPROVED** [review]
Great job Mariya! In the future, I think you should name your branch feat/TT-57_Implement_Translation_Engine. I would say this is a feature and not a conf.

File: fe-traveltrove/13.0.0
Comment (mzolovska) [reply fe-traveltrove/13.0.0:1]: No its now, it was just to push the commit because it wasnt working through the git bash :)

**Pull request merged**